mappings/net/minecraft/util/Util.mapping

```diff
@@ -9,7 +9,7 @@ CLASS net/minecraft/class_156 net/minecraft/util/Util
 	METHOD method_17815 stream (Ljava/util/Optional;)Ljava/util/stream/Stream;
 	METHOD method_17974 ifPresentOrElse (Ljava/util/Optional;Ljava/util/function/Consumer;Ljava/lang/Runnable;)Ljava/util/Optional;
 	METHOD method_18349 getServerWorkerExecutor ()Ljava/util/concurrent/Executor;
```

Comment:
I hope you can fix this one and the fields as well.

Reply:
The name?